lib/storage/sfSessionStorage.class.php

```diff
@@ -68,6 +68,11 @@ public function initialize($options = null)
     // initialize parent
     parent::initialize($options);
 
+    if (!array_key_exists('gc_maxlifetime', $this->options))
+    {
+      $this->options['gc_maxlifetime'] = 1800;
+    }
```

Comment:
This configuration check can be merged on default parameters values used on the first parameter of `array_merge()`, see on line 56.